src/app.ts

```diff
@@ -21,17 +19,21 @@ import https from 'https';
 import WebSocket from 'ws';
 import config from 'config';
 
-if (process.env.NODE_ENV === 'development') {
-    const httpsAgent = new https.Agent({
-        rejectUnauthorized: false,
-    });
+// Manager
```

Comment:
Don't use repetitve comments. If u use self explanatory naming convention, as u have done, is enough